dislib/model_selection/_search.py

```diff
@@ -57,6 +57,10 @@ def fit(self, x, y=None, **fit_params):
 
     def train_candidates(self, x, y, **fit_params):
         """Run fit with all sets of parameters.
+        This method does not generate the scores. By not generating
```

Comment:
No falta una linea en blanco  entre la frase que empieza por Run... y la This method... ?